app/src/main/java/fr/free/nrw/commons/contributions/ContributionController.java

```diff
@@ -25,14 +25,14 @@
 import static fr.free.nrw.commons.contributions.Contribution.SOURCE_GALLERY;
 import static fr.free.nrw.commons.upload.UploadService.EXTRA_SOURCE;
 
-class ContributionController {
+public class ContributionController {
 
```

Comment:
These access modifiers needed to be made public to allow classes in the Nearby package to access them